### PR TITLE
Feature/place componentselector

### LINKED ISSUE #84: Move ComponentSelector into Dialog

### PR DESCRIPTION
**Pull request by ingeridhellen**
## Why is this pull request needed?

This pull request is needed because the ComponentSelector should be inside the Dialog

## What does this pull request change?

* Moves ComponentSelector into the dialog
* Adjust dialog width and styling to fit the new content
* Change order of autocomplete and table inside ComponentSelector

## Issues related to this change:

Closes #84

### CODE REVIEW

File: web/src/components/feature/FluidDialog.tsx [hunk 1]
```diff
@@ -1,28 +1,55 @@
...
+  display: flex;
+  flex-flow: row wrap;
+  gap: 30px;
+`
+
+const FirstColumn = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  gap: 30px;
+`
```
Comment (mariuswinger) [inline web/src/components/feature/FluidDialog.tsx:20]: The styling is not quite right, I think. When selecting "2-2-dimethyl-butane(neohexane) (22-dm-C4)" (the longest component => ComponentSelector becomes wider), the selector jumps beneath the text fields.

File: web/src/components/feature/FluidDialog.tsx [hunk 2]
```diff
@@ -33,6 +60,6 @@ export const FluidDialog = ({
         </Button>
         <Button type="submit">Save</Button>
       </Dialog.Actions>
-    </Dialog>
+    </WideDialog>
```
Comment (mariuswinger) [inline web/src/components/feature/FluidDialog.tsx:63]: I think (some of) the buttons should be on the right side?

Comment (ingeridhellen) [reply web/src/components/feature/FluidDialog.tsx:63]: Yes, but that is outside the scope of this task

**Review by soofstad — APPROVED** [review]
Addition to the other remarks.
Can we make it so the "Select components" component does not have focus on dialog opening?
Point is to not have the drop down open on Dialog opening

Same goes for the drop down for selecting "Fluid package" btw.

**Comment by ingeridhellen** [comment]
> Fluid package



> Addition to the other remarks. Can we make it so the "Select components" component does not have focus on dialog opening? Point is to not have the drop down open on Dialog opening
> 
> Same goes for the drop down for selecting "Fluid package" btw.

Nope, this is a bug in the library used by EDS. However, it should apparently only happen when running in strictmode, which is why it doesn't occur in dev. See https://github.com/equinor/design-system/issues/2369 for details

**Review by mariuswinger — APPROVED** [review]

**Pull request merged**